lib/Bio/EnsEMBL/DataCheck/Checks/CoreBiotypes.pm

```diff
@@ -0,0 +1,103 @@
+=head1 LICENSE
+
+Copyright [2018] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+=cut
+
+package Bio::EnsEMBL::DataCheck::Checks::CoreBiotypes;
+
+use warnings;
+use strict;
+
+use Moose;
+
+use Test::Deep qw{ cmp_deeply };
+use Test::More;
+use Try::Tiny;
+
+extends 'Bio::EnsEMBL::DataCheck::DbCheck';
+
+use constant {  ## no critic 'ProhibitConstantPragma'
+  NAME        => 'CoreBiotypes',
+  DESCRIPTION => 'Checks if the biotype table in core databases has been populated with all the required biotypes',
+  GROUPS      => ['core_handover'],
+  DB_TYPES    => ['core'],
+  TABLES      => ['biotype', 'gene', 'transcript']
+};
+
+
+# Note that this datacheck requires TWO databases to run: a core
+# database to be checked and a Production database containing the
+# master copy of the biotype table. The former is specified the
+# standard way, the latter take advantage of - and possibly slightly
+# abuses - the option -old_server_uri.
+sub tests {
+  my ($self) = @_;
+
+  my $master_dba;
+  try {
+    $master_dba = $self->get_old_dba();
```

Comment:
This is not what `get_old_dba` is for; you need to supply a registry file, and then use `get_dba`, _a la_ https://github.com/Ensembl/ensembl-datacheck/blob/master/lib/Bio/EnsEMBL/DataCheck/Checks/ControlledTablesCore.pm#L41